requirements.txt

```diff
@@ -1,13 +1,10 @@
+git+https://github.com/EGA-archive/crypt4gh.git
 pika==1.0.1
```

Comment:
out of curiosity have you tried updating pika to `1.1.0` ?

Reply:
in setup looks like you don't set version

Reply:
Nope. Haven't tried. We can if you want.

Reply:
as you mentioned not part of this